contracts/core/extensions/CoreInternal.sol

```diff
@@ -98,6 +98,36 @@ contract CoreInternal is
         state.exchanges[_exchangeId] = address(0);
     }
 
+    /**
+     * Add a module address with the mapping of tracked modules.
+     * Can only be called by owner of Core.
+     *
+     * @param _module     Address of the module
+     */
+    function addModule(
+        address _module
+    )
+        external
+        onlyOwner
+    {
+        state.validModules[_module] = true;
```

Comment:
We should emit an event, so we can keep track of history

Reply:
Lets jsut add that to your PR since you have a standard for how you've been doing those.

Reply:
sure